src/components/ListItems/ListItems.js

```diff
@@ -1,20 +1,11 @@
-import React, { Component } from "react";
-
-class ListItems extends Component {
-    createTasks(item) {
-        return <li key={item.key}>{item.text}</li>
-    }
-
-    render() {
-        var Entries = this.props.entries;
-        var listItems = Entries.map(this.createTasks);
-
-        return (
-            <ul>
-                {listItems}
-            </ul>
-        );
-    }
-}
+import React from 'react';
+
+const ListItems = ({ entries }) => (
+  <ul>
+    {entries.map(item => (
+      <li key={item.key}>{item.text}</li>
+    ))}
+  </ul>
+);
```

Comment:
In its most basic form a functional component is just the render function from a class component.